src/Select/Select.less

```diff
@@ -4,103 +4,114 @@
 
```

Comment:
I recommend reviewing the changes to this file commit by commit:

https://github.com/Clever/components/pull/125/commits/0d7d8bc28b075f485d1db467aa703649234c19ad Actually updates the stylesheet's contents.
https://github.com/Clever/components/pull/125/commits/a75f0c954f30f4b40c650debd219f70bb4fefadd Removes style nesting where (easily) possible.